DMD2_Timer.cpp

```diff
@@ -35,6 +35,13 @@
 
 #define ESP8266_TIMER0_TICKS microsecondsToClockCycles(250) // 250 microseconds between calls to scan_running_dmds seems to works better than 1000.
 
+// Assuming esp32 using frequency of 80MHz...
+#define ESP32_TIMER0 0
+// run counter to increment every 1 microseconds
+#define ESP32_TIMER0_PRESCALER 80
+// call scan_running_dmds every 250 microseconds on ESP32
```

Comment:
250 ms is too fast. changing back to 1000

Reply:
Interrupts need IRAM_ATTR functions, so this is definitely not gonna work. One solution is to use RTOS task scheduling, but I don't have the resources to test it now since my project is already done. I was using beginNoTimer() and call scanDisplay() in loop function.

Reply:
Hi, I have found your ESP32 port and I am attempting to use it to drive a DMD with my ESP32, but I can not find the complete wiring. Looking in the code, I found these: pin_noe == 14 && pin_a == 27 && pin_b == 26 && pin_sck == 25
But the panel also has CLK and R(1) pins. Where do those go?
Thank you very much in advance!

Reply:
I'm sure CLK and R(1) are SPI pins. R(1) is the data pin if you want to use red led and B(1) is for blue led (if any).

Reply:
I found the 250ms is good for one panel if you want to use more than one panel you need to change this to 500ms or higher depending on how many screens.